python_modules/dagster/dagster_tests/definitions_tests/module_loader_tests/test_load_assets_from_modules.py

```diff
@@ -178,6 +178,17 @@ def top_level_spec():
         ):
             load_assets_from_modules([asset_package, module_with_assets], include_specs=True)
 
+    # Create an asset spec with an identical key to a full fledged assetsdef. Won't cause an error unless include_specs=True
```

Comment:
add new tests from https://github.com/dagster-io/dagster/pull/27025/files as well?

Reply:
can be in a follow up